docs/source/cheat_sheet_py3.rst

```diff
@@ -105,6 +105,45 @@ Python 3 introduces an annotation syntax for function declarations in `PEP 3107
        
        ...
 
+Coroutines and asyncio
+**********************
+
+See :ref:`async-and-await` for the full detail on typing coroutines and asynchronous code.
+
+.. code-block:: python
+
+   import asyncio
+   from typing import Generator, Any
+
+   # A Python 3.4 coroutine should have a return type of
```

Comment:
I think it would be better to say a "generator based coroutine" or "async generator" here instead of `Python 3.4 coroutine` which is a bit ambiguous.